### PR TITLE
WRQ-206: Made `--no-animation` public and added documentation

### PR DESCRIPTION
**Pull request by hong6316**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
The`--no-animation` pack option has merged as private API in #316 .
Now this option will be made as public.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
- Made `--no-animation` public 
- Added documentation

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRQ-206

### Comments
Enact-DCO-1.0-Signed-off-by: Taeyoung Hong (taeyoung.hong@lge.com)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## unreleased
+
+* Switched `--no-animation` as a public option.
```
Comment (MikyungKim) [inline CHANGELOG.md:3]: Please write the changelog correctly.
Whrere is the category? Is it "Switched"?

Comment (MikyungKim) [inline CHANGELOG.md:3]:
```suggestion
### pack

* Added `--no-animation` option to build without effects such as animation and shadow for low performant devices.
```

File: docs/building-apps.md [hunk 2]
```diff
@@ -224,6 +225,11 @@ my-app/
   webos-meta/
 ```
 
+## Turn off animations
+
+For low-performant devices, Enact CLI provides `--no-animation` option to remove animations and graphical effects such as shadow.
+Animation will be turned off with this build options, without modifying the app code.
```
Comment (MikyungKim) [inline docs/building-apps.md:231]:
```suggestion
## Build without Effects

To accommodate devices with lower performance, the Enact CLI offers the `--no-animation` option. This option disables animations and graphical effects, including shadows. When activated, it sets the `ENACT_PACK_NO_ANIMATION` environment variable. This variable allows UI libraries like Sandstone to conditionally disable effects. Additionally, you can leverage this variable in your application to achieve the same outcome. Thus, you can develop an app devoid of these effects and do so without modifying your codebase.
```

File: CHANGELOG.md (hunk 1 above)
Comment (hong6316) [reply CHANGELOG.md]: OK

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,9 @@
+## unreleased
+
+### pack
+
+* Added `--no-animation` option to build without effects such as animation and shadow for low performant devices.
```
Comment (MikyungKim) [inline CHANGELOG.md:5]:
```suggestion
* Added `--no-animation` option to build without effects such as animation and shadow.
```

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**